models/QtiRunnerInitDataBuilder.php

```diff
@@ -43,6 +45,9 @@ class QtiRunnerInitDataBuilder
 {
     protected const OUTCOME_VAR_SCORE = 'SCORE';
     protected const OUTCOME_VAR_MAXSCORE = 'MAXSCORE';
+    private const OUTCOME_DECLARATION_IDENTIFIER = 'identifier';
+    private const OUTCOME_DECLARATION_MAXSCORE = 'MAXSCORE';
```

Comment:
`OUTCOME_VAR_MAXSCORE` constant seems to be here to represent exactly the same value.
I guess it can be reused.